web/template/layout/header.gohtml

```diff
@@ -1,45 +1,91 @@
-                  <div class="govuk-form-group">
-                    <label class="govuk-label moj-search__label govuk-visually-hidden" for="search">
-                      Search
-                    </label>
-                    <input class="govuk-input moj-search__input" id="search" name="term" type="search">
-                  </div>
+    </header>
 
-                  <button class="govuk-button app-!-moj-search__button" data-module="govuk-button">
-                    <span class="govuk-visually-hidden">Search</span>
-                    <svg viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="moj-search-toggle__button__icon govuk-!-margin-0" ><path d="M7.433,12.5790048 C6.06762625,12.5808611 4.75763941,12.0392925 3.79217348,11.0738265 C2.82670755,10.1083606 2.28513891,8.79837375 2.28699522,7.433 C2.28513891,6.06762625 2.82670755,4.75763941 3.79217348,3.79217348 C4.75763941,2.82670755 6.06762625,2.28513891 7.433,2.28699522 C8.79837375,2.28513891 10.1083606,2.82670755 11.0738265,3.79217348 C12.0392925,4.75763941 12.5808611,6.06762625 12.5790048,7.433 C12.5808611,8.79837375 12.0392925,10.1083606 11.0738265,11.0738265 C10.1083606,12.0392925 8.79837375,12.5808611 7.433,12.5790048 L7.433,12.5790048 Z M14.293,12.579 L13.391,12.579 L13.071,12.269 C14.2300759,10.9245158 14.8671539,9.20813198 14.866,7.433 C14.866,3.32786745 11.5381325,-1.65045755e-15 7.433,-1.65045755e-15 C3.32786745,-1.65045755e-15 -1.65045755e-15,3.32786745 -1.65045755e-15,7.433 C-1.65045755e-15,11.5381325 3.32786745,14.866 7.433,14.866 C9.208604,14.8671159 10.9253982,14.2296624 12.27,13.07 L12.579,13.39 L12.579,14.294 L18.296,20 L20,18.296 L14.294,12.579 L14.293,12.579 Z"></path></svg>
-                  </button>
-                </form>
+    <div class="govuk-width-container">
+      <div class="govuk-grid-row govuk-phase-banner">
+        <div class="govuk-grid-column-two-thirds">
+          <p class="govuk-phase-banner__content">
+            <strong class="govuk-tag govuk-phase-banner__content__tag">
+              Alpha
+            </strong>
+            <span class="govuk-phase-banner__text">
+              This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.
+            </span>
+          </p>
+        </div>
+
+        <div class="govuk-grid-column-one-third app-search-inline-phase-banner">
+          <div class="moj-search moj-search--ondark moj-search--inline">
+            <form class="form" method="get" action="{{ prefix "/search" }}" data-module="search">
+              <div class="govuk-form-group">
+                <label class="govuk-label moj-search__label govuk-visually-hidden" for="search">
+                  Search
+                </label>
+                <input class="govuk-input moj-search__input app-moj-search__input" id="search" name="term" type="search" placeholder="Search for a case">
```

Comment:
Both search boxes have `id="search"`, which is invalid, but I don't think either of them actually need an ID so you can get rid of them both.
```suggestion
                <input class="govuk-input moj-search__input app-moj-search__input" name="term" type="search" placeholder="Search for a case">
```